src/appmixer/twilio/calls/NewCall/NewCall.js

```
@@ -20,7 +20,8 @@
 
         } else if (context.messages.webhook) {
             // Twilio HTTP callback request.
-            return context.sendJson(context.messages.webhook.content.data, 'call');
+            await context.sendJson(context.messages.webhook.content.data, 'call');
+            return context.response(); 
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)